charts/argo/values.yaml

```diff
@@ -71,6 +71,8 @@ ui:
 useDefaultArtifactRepo: false
 useStaticCredentials: true
 artifactRepository:
+  # archiveLogs will archive the main container logs as an artifact
+  archiveLogs: true
```

Comment:
Should this be false by default?

Reply:
When some enables `useDefaultArtifactRepo` wouldn't make sense to automatically start archiving the logs ?

Reply:
For instance, if `minio.install` is enabled, `minio.defaultBucket.enabled` is automatically enabled by default

Reply:
Not sure. Does this change behaviour?

Reply:
I'm not sure. Probably only for those who have already have enabled `useDefaultArtifactRepo`, now it will automatically upload logs as well. Should we just disable it by default?

Reply:
I think so. Better not to have surprises when you upgrade.

Reply:
Agreed. I've just updated the PR and set `archiveLogs` to `false` by default.